lib/presentation/widget/upload_request_group/created/created_upload_request_group_widget.dart

```diff
@@ -0,0 +1,221 @@
+        return !appState.uiState.isInSearchState()
+            ? OrderByButtonBuilder(context, appState.createdUploadRequestGroupState.pendingSorter)
+                .onOpenOrderMenuAction(
+                    (currentSorter) => _model.openPopupMenuSorterPending(context, currentSorter))
+                .build()
+            : SizedBox.shrink();
+      },
+    );
+  }
+
+  Widget _buildListData(List<SelectableElement<UploadRequestGroup>> uploadRequestList) {
+    if (uploadRequestList.isEmpty) {
+      return _buildCreateUploadRequestsHere(context);
+    } else {
+      return ListView.builder(
+          itemCount: uploadRequestList.length, 
+					itemBuilder: (context, index) => 
+						_buildPendingUploadTile(uploadRequestList[index]));
+    }
+  }
+
+  Widget _buildCreateUploadRequestsHere(BuildContext context) {
+    return BackgroundWidgetBuilder()
+        .key(Key('create_upload_request_here'))
+        .image(SvgPicture.asset(imagePath.icCreateUploadRequest,
+            width: 120, height: 120, fit: BoxFit.fill))
+        .text(AppLocalizations.of(context).create_upload_requests_here)
+        .build();
+  }
+
+  ListTile _buildPendingUploadTile(SelectableElement<UploadRequestGroup> request) {
+    return ListTile(
+        onTap: () {
+          _model.getListUploadRequests(request.element);
+        },
+        dense: true,
+        leading: _getTileIcon(request.element.collective),
+        title: Text(request.element.label, style: TextStyle(fontSize: 14, color: AppColor.uploadRequestLabelsColor)),
+        subtitle: Text(AppLocalizations.of(context).activated_date(request.element.activationDate.getMMMddyyyyFormatString()),
+            style: TextStyle(fontSize: 14, color: AppColor.uploadFileFileSizeTextColor)),
+        trailing: IconButton(
+            icon: SvgPicture.asset(
+              imagePath.icContextMenu,
+              width: 24,
+              height: 24,
+              fit: BoxFit.fill,
+            ),
+            onPressed: () => {}),
+        onLongPress: () => {});
+  }
+
+  Widget _getTileIcon(bool collective) {
+    return Column(mainAxisAlignment: MainAxisAlignment.center, children: [
+      SvgPicture.asset(
+          collective ? imagePath.icUploadRequestCollective : imagePath.icUploadRequestIndividual,
+          width: 20,
+          height: 24,
+          fit: BoxFit.fill)
+    ]);
+  }
+
+  Widget _buildSearchViewWidget(BuildContext context) {
+		return StoreConnector<AppState, CreatedUploadRequestGroupState>(
+			converter: (store) => store.state.createdUploadRequestGroupState,
+			builder: (_, state) {
+				if (_model.searchQuery.value.isEmpty) {
+					return SizedBox.shrink();
+				}
+				if (state.uploadRequestsCreatedList.isEmpty) {
+					return _buildNoResultFound();
+				}
+				return Column(children: [
+					_buildResultCount(state.uploadRequestsCreatedList.length),
+					Expanded(
+						child: _buildListData(state.uploadRequestsCreatedList))
+				]);
+			});
+  }
```

Comment:
indent

Reply:
I don't know why but it is good indent in my local machine.

<img width="791" alt="Screen Shot 2021-08-23 at 11 35 06" src="https://user-images.githubusercontent.com/29337364/130390496-ecdadf41-38ac-404d-ae24-34f49ac26c73.png">